integration_tests/cli/validator-demo.sh

```diff
@@ -406,59 +421,197 @@ echo "$result"
-result=$(docker exec "$container" /bin/sh -c "echo test1234 | dcld query validator disabled-node --address=$validator_address")
-check_response "$result" "\"creator\": \"$alice_address\""
-check_response "$result" "\"disabledByNodeAdmin\": false"
+echo "Get a proposed validator to disable $address"
+result=$(docker exec "$container" /bin/sh -c "echo test1234 | dcld query validator proposed-disable-node --address="$address"")
+check_response "$result" "\"approvals\":\[{\"address\":\"$alice_address\"" raw
+check_response "$result" "\"rejects\":\[{\"address\":\"$bob_address\"" raw
 check_response "$result" "\"address\": \"$validator_address\""
 echo "$result"
 
 test_divider
 
+echo "Get a proposed validator to disable $validator_address"
+result=$(docker exec "$container" /bin/sh -c "echo test1234 | dcld query validator proposed-disable-node --address="$validator_address"")
+check_response "$result" "\"approvals\":\[{\"address\":\"$alice_address\"" raw
+check_response "$result" "\"rejects\":\[{\"address\":\"$bob_address\"" raw
+check_response "$result" "\"address\": \"$validator_address\""
+echo "$result"
+
+test_divider
+
+echo "Jack rejects to disable validator $address"
+result=$(dcld tx validator reject-disable-node --address="$validator_address" --from jack --yes)
+check_response "$result" "\"code\": 0"
+echo "$result"
+
+test_divider
+
+echo "Jack doesn't reject to disable validator $address, because Jack already rejected to disable validator"
+result=$(dcld tx validator reject-disable-node --address="$validator_address" --from jack --yes)
+response_does_not_contain "$result" "\"code\": 0"
+echo "$result"
+
+test_divider
+
+echo "Jack doesn't approves to disable validator $address, because Jack already rejected to disable validator"
+result=$(dcld tx validator approve-disable-node --address="$validator_address" --from jack --yes)
+response_does_not_contain "$result" "\"code\": 0"
+echo "$result"
+
+
+test_divider
+
+echo "Get all proposed validators to disable. $address in the list"
+result=$(docker exec "$container" /bin/sh -c "echo test1234 | dcld query validator all-proposed-disable-nodes")
+response_does_not_contain "$result" "\"approvals\":\[{\"address\":\"$alice_address\"" raw
+response_does_not_contain "$result" "\"rejects\":\[{\"address\":\"$bob_address\"" raw
+response_does_not_contain "$result" "\"address\": \"$validator_address\""
+echo "$result"
+
+test_divider
+
+echo "Get all rejected validators to disable. $address in the list"
+result=$(docker exec "$container" /bin/sh -c "echo test1234 | dcld query validator all-rejected-disable-nodes")
+check_response "$result" "\"approvals\":\[{\"address\":\"$alice_address\"" raw
+check_response "$result" "\"rejects\":\[{\"address\":\"$bob_address\"" raw
+check_response "$result" "\"address\": \"$jack_address\""
+check_response "$result" "\"address\": \"$validator_address\""
+echo "$result"
+
+test_divider
+
+echo "Get a proposed validator to disable $address"
```

Comment:
Inconsistent message. Since rejected disable node is queried

Reply:
Done! ✅